### DISCUSSION TITLE
Pimcore 11 Datahub config not showing up

### DISCUSSION THREAD

**Discussion by WillemAlbert** [pimcore/pimcore#15564] [General]
I use the community edition V 11.0.3 and have installed
Datahub and Data Importer conform the instructions

<img width="1103" alt="Screenshot 2023-07-19 at 15 56 31" src="https://github.com/pimcore/pimcore/assets/45628579/e7f08ad6-9a71-4c65-9c99-9dd1f03a724f">

I do not see Datahub config. I am logged in as an admin, the screen looks like this:

![Screenshot 2023-07-19 at 15 53 43](https://github.com/pimcore/pimcore/assets/45628579/9d46cf7b-9e51-4f94-b631-c9575155a0b4)

I am able to create a GraphQL, the tabs are there
I am *not* able to create a Data Objects Importer, no tabs.

I have been at it for hours, reading docs and googling did not help.

Anyone any idea how to fix this?

**Comment by WillemAlbert** [comment]
To clarify "no tabs" Here are two screenshots:

GraphQL with tabs

![Screenshot 2023-07-22 at 13 53 51](https://github.com/pimcore/pimcore/assets/45628579/cd9dfe49-639b-435b-8f4b-61c014921f24)

Data Objects Importer WITHOUT tabs:

![Screenshot 2023-07-22 at 13 54 02](https://github.com/pimcore/pimcore/assets/45628579/c3859f1c-f516-4a4b-8afe-34510a74c77e)

Also GraphQL doe not work but there already is a bug report from someone else: 
https://github.com/pimcore/pimcore/issues/15502

**Comment by johanlindgren** [comment]
Check your browser console, it probably complains about missing PimcoreApplicationLogger?

Try to install PimcoreApplicationLogger 

```
php bin/console pimcore:bundle:install PimcoreApplicationLoggerBundle
```

and put it early in your bundles.php 
```
<?php
return [
     Pimcore\Bundle\ApplicationLoggerBundle\PimcoreApplicationLoggerBundle::class => ['all' => true],
     Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle::class => ['all' => true],
     Pimcore\Bundle\DataImporterBundle\PimcoreDataImporterBundle::class => ['all' => true],
];
```

I guess DataImporter and DataHub should register ApplicationLogger with higher priority, but the above solves the problem

**Reply by WillemAlbert** [reply]
SOLVED!

I already installed the software but the entry in bundles.php was missing.

Thank you for your reply, it's much appreciated!
